assets/src/apps/authoring/icons/icon-multiChoice.svg

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="29px" height="24px" viewBox="0 0 29 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>icon-multiChoice</title>
+    <g id="Screen-Panel-/-Adaptivity" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
```

Comment:
I see this same id in another file, are the ids even necessary?

Reply:
they look like export artifacts from the design program i used. probably safe to remove, but might want to test.

something i forgot to do with these svgs is run them thru an optimizer like: https://jakearchibald.github.io/svgomg/

Reply:
I checked after removing the id and the things were looking fine.